src/reactblockui.d.ts

```diff
@@ -8,7 +8,7 @@ export type BlockUiProps<T = {}> = React.HTMLAttributes<HTMLElement> & {
     className?: string,
     message?: string | React.ReactNode,
     loader?: ((...args: any[]) => any) | string | React.ReactNode,
-    tag?: (...args: any[]) => any | string,
+    tag?: ((...args: any[]) => any) | string,
```

Comment:
I think we can just replace the function with `React.ReactType` correct?

Reply:
Okay replaced it